.github/workflows/ci.yml

```diff
@@ -391,6 +391,12 @@ jobs:
 
   pkg-preview:
     name: Pkg Preview
-    # needs:
-    #   - cargo-deny
+    needs:
+      - test-linux
+      - test-windows
+      - cargo-deny
+      - lint
+      - rust_check
+      - rust_test
```

Comment:
Is this automatically triggered in all mrs?

`rust_test` needs `rust_changes`, so `pkg-preview` is skipped ?